app/services/builders/chart_summary/year_to_date_blog_visits.rb

```diff
@@ -3,8 +3,8 @@ module ChartSummary
     class YearToDateBlogVisits < BaseService
       def call
         Metric
-          .where(name: Metric.names[:blog_visits], ownable_type: Technology.to_s)
-          .where('value_timestamp > ?', Time.zone.now.beginning_of_year)
+          .where(name: Metric.names[:blog_visits], ownable_type: BlogPost.to_s)
```

Comment:
just curious, but what is this change doing in terms of UI? 

Do we need to reconcile all existing data so that the ownable is now the blog post and not the technology?

Reply:
The problem is that now that a single blog post may belong to more than one technology, its number of visits may be duplicated across those technologies.
For instance, imagine having a single blog post, with both `ruby` and `react` technologies. If that blog post receives 100 visits in any month, both `ruby` and `react` will have 100 visits each. So when I want to get the total number of visits across all technologies, if I just add the visits for each technology I'll end up with 200, which is false. That's why I need to add the data directly from the blog posts visits.

Reply:
Makes sense. And do we need to update existing records? As far as I understand from this code, metrics used to have the technology as the ownable, and now they'll have the blog post, right?

Reply:
The best part is that the visits metrics for blog posts are already there. We create visits metrics for both blog posts and technologies so there's no need to update any records because of that 😎 
But of course, we will update the blog posts associated technologies, so the metrics for technologies will have to be updated as well. It's not a big problem, though, we just need to run 2 jobs (one that updates the blog posts and another that updates the metrics). Both jobs are already created, so no further PRs are needed.